packages/fiori/test/specs/ViewSettingsDialog.spec.js

```diff
@@ -0,0 +1,102 @@
+const assert = require("chai").assert;
+const PORT = require("./_port.js");
+
+describe("ViewSettingsDialog general interaction", () => {
+	before(() => {
+		browser.url(`http://localhost:${PORT}/test-resources/pages/ViewSettingsDialog.html`);
+	});
+
+	it("test ViewSettingsDialog initial value", ()=>{
+		const btnOpenDialog = $("#btnOpenDialog");
+		const viewSettingsDialog = browser.$("#vsd");
+		btnOpenDialog.click();
+
+		assert.strictEqual(viewSettingsDialog.shadow$("ui5-list").$("ui5-li[selected]").getText(), "Ascending", "initially sortOrder has correct value");
+		assert.notOk(viewSettingsDialog.$("ui5-li[selected]").isExisting(), "initially sortBy should not have an option selected");
+
+		browser.keys("Escape");
+	});
+
+	it("test ViewSettingsDialog - sortOrder confirm selected settings", ()=>{
+		const btnOpenDialog = $("#btnOpenDialog");
+		const viewSettingsDialog = browser.$("#vsd");
+		btnOpenDialog.click();
+
+		viewSettingsDialog.shadow$("ui5-list").$$("ui5-li")[1].click();
+
+		viewSettingsDialog.shadow$("ui5-dialog").$(".ui5-vsd-footer").$("ui5-button").click();
+
+		btnOpenDialog.click();
+
+		assert.strictEqual(viewSettingsDialog.shadow$("ui5-list").$("ui5-li[selected]").getText(), "Descending", "SortOrder should properly change value");
+
+		browser.keys("Escape");
+	});
+
+	it("test ViewSettingsDialog - sortBy confirm selected settings", ()=>{
+		const btnOpenDialog = $("#btnOpenDialog");
+		const viewSettingsDialog = browser.$("#vsd");
+		btnOpenDialog.click();
+
+		assert.notOk(viewSettingsDialog.$("ui5-li[selected]").isExisting(), "sortBy should not have an option selected");
+
+		viewSettingsDialog.$$("ui5-li")[0].click();
+		viewSettingsDialog.shadow$("ui5-dialog").$(".ui5-vsd-footer").$("ui5-button").click();
+
+		btnOpenDialog.click();
+
+		assert.strictEqual(viewSettingsDialog.$("ui5-li[selected]").getText(), "Name", "sortBy should  have an option selected");
+
+		browser.keys("Escape");
+	});
+
+	it("test ViewSettingsDialog change selected settings", ()=>{
+		const btnOpenDialog = $("#btnOpenDialog");
+		const viewSettingsDialog = browser.$("#vsd");
+		btnOpenDialog.click();
+
+		assert.strictEqual(viewSettingsDialog.$("ui5-li[selected]").getText(), "Name", "sortBy should have an option selected");
+
+		viewSettingsDialog.$$("ui5-li")[1].click();
+		viewSettingsDialog.shadow$("ui5-dialog").$(".ui5-vsd-footer").$("ui5-button").click();
+		btnOpenDialog.click();
+
+		assert.strictEqual(viewSettingsDialog.$("ui5-li[selected]").getText(), "Position", "sortBy should be change selected option");
```

Comment:
I suppose this should be: sortBy should change selected option

Reply:
Done